Sources/Sentry/SentryURLProtocolTracker.m

```diff
@@ -0,0 +1,87 @@
+#import "SentryURLProtocolTracker.h"
+#import "SentryPerformanceTracker.h"
+
+@interface
+SentryURLProtocolTracker () <NSURLSessionDelegate>
+
+@property (nonatomic, strong) NSURLSession *session;
+
+@property (nonatomic, strong) NSString *spanId;
+@property (nonatomic, strong) NSString *parentSpanId;
+@end
+
+@implementation SentryURLProtocolTracker
+
++ (BOOL)canInitWithRequest:(NSURLRequest *)request
+{
+    return ![request.URL.host.lowercaseString containsString:@"sentry"]
+        && ([request.URL.scheme isEqualToString:@"http"] ||
+            [request.URL.scheme isEqualToString:@"https"]);
```

Comment:
why do we need to do that? to not span sentry calls?

Reply:
maybe we should filter if its the DSN URL instead of that, so we solve this problem